packages/components/src/components/select/select.lite.tsx

```diff
@@ -182,7 +182,7 @@ export default function DBSelect(props: DBSelectProps) {
 
 			if (stringPropVisible(props.message, props.showMessage)) {
 				state._descByIds = messageId;
-			} else {
+			} else if (props.placeholder) {
 				state._descByIds = placeholderId;
 			}
```

Comment:
```suggestion
			} else if (props.placeholder) {
				state._descByIds = placeholderId;
			} else {
				state._descByIds = undefined;
			}
```

we would still need a default case.